### PR TITLE
Add a code sample to the readme

### PR DESCRIPTION
**Pull request by mdiep**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
+            print("launched task: \(task)")
+
+        case let .StandardError(data):
+            print("stderr: \(data)")
+
+        case let .StandardOutput(data):
+            print("stdout: \(data)")
+
+        case let .Success(string):
+            print("value: \(string)")
```
Comment (ikesyo) [inline README.md]: Should we do the same here as above?

```swift
print("value: \(string ?? "")")
```

**Comment by ikesyo** [comment]
This should help the new commers! :sparkling_heart:

**Comment by mdiep** [comment]
:sparkles:

**Comment by ikesyo** [comment]
:shipit:

**Pull request merged**